File: ueueo-framework/ueueo-localization/src/main/java/Abp/Localization/StaticLocalizationDictionary.cs
using System.Collections.Generic;
using Microsoft.Extensions.Localization;

namespace Volo.Abp.Localization;

/**
 * Represents a simple implementation of <see cref="ILocalizationDictionary"/> interface.
*/
public class StaticLocalizationDictionary : ILocalizationDictionary
{

    public String CultureName;//  { get; }

    protected Dictionary<String, LocalizedString> Dictionary;//  { get; }

    /**
     * Creates a new <see cref="StaticLocalizationDictionary"/> object.
    *
     * <param name="cultureName">Culture of the dictionary</param>
     * <param name="dictionary">The dictionary</param>
     */
    public StaticLocalizationDictionary(String cultureName, Dictionary<String, LocalizedString> dictionary)
    {
        CultureName = cultureName;
        Dictionary = dictionary;
    }


    public   LocalizedString GetOrNull(String name)
    {
        return Dictionary.GetOrDefault(name);
    }

    public void Fill(Dictionary<String, LocalizedString> dictionary)
    {
        for (var item in Dictionary)
        {
            dictionary[item.Key] = item.Value;
        }
    }
}

File: ueueo-framework/ueueo-localization/src/main/java/com/ueueo/localization/StaticLocalizationDictionary.java
package com.ueueo.localization;

import lombok.Data;

import java.util.Map;

/**
 * Represents a simple implementation of <see cref="ILocalizationDictionary"/> interface.
 */
@Data
public class StaticLocalizationDictionary implements ILocalizationDictionary {

    private String cultureName;

    private Map<String, LocalizedString> dictionary;

    /**
     * Creates a new <see cref="StaticLocalizationDictionary"/> object.
     *
     * <param name="cultureName">Culture of the dictionary</param>
     * <param name="dictionary">The dictionary</param>
     */
    public StaticLocalizationDictionary(String cultureName, Map<String, LocalizedString> dictionary) {
        this.cultureName = cultureName;
        this.dictionary = dictionary;
    }

    @Override
    public LocalizedString getOrNull(String name) {
        return dictionary.get(name);
    }

    @Override
    public void fill(Map<String, LocalizedString> dictionary) {
        dictionary.putAll(this.dictionary);
    }

}
